services/collection/create_index.cpp

```diff
@@ -44,7 +44,8 @@ namespace services::collection {
     void collection_t::create_index(const session_id_t& session, create_index_t& index) {
         debug(log(), "collection::create_index : {} {} {}", name_.to_string(), name_index_type(index.index_type_), keys_index(index.keys_)); //todo: maybe delete
         if (dropped_) {
-            actor_zeta::send(current_message()->sender(), address(), handler_id(route::create_index_finish), session, result_create_index(false));
+            actor_zeta::send(current_message()->sender(), address(), handler_id(route::create_index_finish), session,
+                             make_cursor(actor_zeta::detail::pmr::get_default_resource(), error_code_t::collection_dropped));
```

Comment:
@OrangeSmokingJacket @kotbegemot sorry I don't get, for actors we use `actor_zeta::detail::pmr::get_default_resource()` in `make_cursor` ? Not `std::prm::get_default_resource()` ?